release_notes_generator/model/custom_chapters.py

```diff
@@ -33,11 +33,11 @@ def populate(self, records: dict[int, Record]):
         :param records: A dictionary of records where the key is an integer and the value is a Record object.
         """
         for nr in records:                                      # iterate all records
-            for ch_key in self.chapters:                        # iterate all chapters
+            for _, ch in self.chapters.items():                 # iterate all chapters
```

Comment:
I would use chapters.values() instead.

Reply:
The usage of items was a proposal from Pylint. Update to values().